external/fv3fit/fv3fit/emulation/transforms/transforms.py

```diff
@@ -71,8 +71,37 @@ def backward(self, x: tf.Tensor) -> tf.Tensor:
         return tf.math.exp(x)
 
 
+@dataclasses.dataclass
+class PositiveTransform:
+    """
+    A univariate transformation for::
+
+    y := x
+    x := ReLU(y)
+
+    Attributes:
+        enforce_positive: used only for identifying this transform in the
+            configuration
+    """
+
+    # TODO: Is there a better way to do config instance via dacite w/o uniqueness?
+    enforce_positive: str = "yes_please"
```

Comment:
Don't use `enforce_positive`, but I need something to differentiate it for instantiation via `dacite`.  Know a better way to handle it while maintaining the nice type-based dispatch?

Reply:
Hmm. if we keep going down this road we'll need name based dispatch...but that starts to feel like even more overengineering. Maybe would be more attractive to make this a boolean or generalize the class to limit the outputs between `upper` and `lower` limits.

Reply:
> generalize the class to limit the outputs between upper and lower limits.

I like this. I'll give it a shot.